TEMPLATE.md

```diff
@@ -0,0 +1,87 @@
+
+
+
+## Introduction
+
+This section should detail the following: A summary of the proposal’s
+motivation, implementation, risks, timeline, and goals. It should aim to
+be an abstract for the proposals.
+
+## Background
+
+This section should explain any background (targeting a casual audience)
+needed to understand the proposal’s motivation (e.g. a high level
+overview of the technical details and some history).
+
+
+
+## Motivation
+
+This section should explain the motivation in terms of why Haskell
+Foundation should take on this project, and why it is important for the
+Haskell ecosystem. What impact will it have? Why should people care?
+
+## Goals
+
+This section should detail the high-level goals of the project, as well
+as the criteria for what constitutes a finished project. Deliverables
+should support these goals, and the goals should support the motivation
+for undertaking the project.
+
+## People
+
+This section should detail the following people:
+
+-   **Performers:** the project leads and main developers/resources who will be directly involved in the day to day work.
+    
+-   **Reviewers:** people who will be signing off on work done for the proposal at different stages.
+    
+-   **Stakeholders:** A sample of people who will be directly affected by the project, who can serve as coordinators and advisors.
+
+## Resources
+
+This section should describe the estimated resource cost of the project
+as it pertains to the Haskell Foundation. Does the project need funding?
+Is there a human cost in terms of hours and effort? We need to know!
+
+## Timeline
+
+This section should detail the high-level time estimate of such a
+project, and serve as an educated guess. We acknowledge that no estimate
+is accurate, but we should be able to give coarse estimates in terms of
+months and years.
+
+## Lifecycle:
+
+This section should detail the SDLC of the project, including a
+high-level description of the following criteria:
+
+-   **Implementation:** A high-level overview of the implementation details of the project, including how the leaders expect to make progress on individual goals and key progress indicators. This
+    section should not be as granular as individual merge requests that will be made, but that would be a bonus!
+    
+-   **Compatibility issues:** highlight any potential compatibility issues that may arise from the changes brought by this project, including any work that needs to be done on the part of
+    stakeholders or performers.
+    
+-   **Performance impact:** An overview of the potential performance impact of this project on the ecosystem, if applicable.
+
+## Deliverables
+
+This section should describe the individual deliverables this project
+will bring as a result of the changes described above. These
+deliverables should support the end goals of the project and define what
+HF is actually delivering to the community.
```

Comment:
```suggestion
this project is actually delivering to the community.
```

The HF might deliver organization, for example, while the project delivers a new wizwoz to be uploaded to Hackage.